models/components/versions/base/component.go

```diff
@@ -19,4 +19,9 @@ type Satisfies interface {
 	GetControlKey() string
 	GetNarrative() string
 	GetCoveredBy() common.CoveredByList
+}
+
+// Base is the common struct that all component schemas must have.
```

Comment:
Not a blocker, but I wonder if there's a Go-type-system-friendly way to enforce this.